### ISSUE TITLE
CommentArray.sort() returns undefined

### ISSUE THREAD

**Issue by Jason3S** [kaelzhang/node-comment-json#32]
**Tell us about your environment**

* **Node Version:**
  Node 12.x, 14.x, 16.x
* **comment-json Version**
  4.1.0
**Please show your use case / code slices / code link that could reproduce the issue**

```typescript
import { parse } from 'comment-json';

describe('Validate comment-json', () => {
    test('CommentArray', () => {
        const s = {
            words: ['one', 'two', 'three'],
        };

        const t = parse(JSON.stringify(s, null, 2));

        const expected = s.words.concat(['four']).sort();
        const actual = t['words'].concat(['four']).sort();
        expect(actual).toEqual(expected);
    });
});
```

**What did you expect to happen?**
  To get back the sorted array.
**Are you willing to submit a pull request to fix this bug?**
  yes

**Comment by kaelzhang** [comment]
Landed in `4.1.1`

**Comment by Jason3S** [comment]
Thank you very much!
